clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -609,6 +609,26 @@ public void testOpen() throws IOException, ApiException {
         }
     }
 
+    @Test
+    public void testOpenWithSpace() throws IOException, ApiException {
+        String contents = "The quick brown fox jumps over the lazy dog.";
+        byte[] contentsBytes = contents.getBytes();
+        String physicalKey = "/repo-base/with space/open";
```

Comment:
Can we also test some other bad characters, please?  Maybe try `%$#;:&?` (tests known-awful HTTP characters), `אל/הקובץ` (tests Unicode and all-important Hebrew support) and even a Unicode emoji `🤯` U+1F92F (tests Unicode crazy pages (not a technical term), probably OK if it fails but great if it doesn't).